### DISCUSSION TITLE
Are the recommendations for `targets` x `shiny` still valid for the new release?

### DISCUSSION THREAD

**Discussion by cpauvert** [wlandau/targets-shiny#12] [General]
Hi @wlandau,
Thanks for creating and maintaining `{targets}`! I am a new user and am planning to use  `{targets}` with `{shiny}` as you showcased in this repository.

I was wondering whether if you consider that the recommendations listed in this repository (regarding persistent storage for instance) still holds with the recent releases of  `{targets}`?

Indeed, the documentation here mentions an early release of `{targets}` as well as features in development:

https://github.com/wlandau/targets-shiny/blob/00e792e5c05ca579c0550cc758ca11a972aa9fe4/README.md?plain=1#L16

With the new release (1.3.0) of `{targets}` on the CRAN, I was wondering if you would do things differently for the integration of `{shiny}` and `{targets}`?

Best,

**Comment by wlandau** [comment]
The data/metadata cloud storage capabilities in `targets` 1.3.0 will generally be a better fit that local storage: https://books.ropensci.org/targets/cloud-storage.html. However, in cases when cloud storage is not desired or feasible, I think the general pattern in this repo is still useful.

**Reply by cpauvert** [reply]
That was quick, thanks! Ok noted! Is the mechanism for cloud storage in `targets` generic enough to use custom S3 bucket? Say from academic institutions? If this question is too `targets` specific and should be on the adequate repository, let me know.

**Reply by wlandau-lilly** [reply]
Yes, custom buckets and endpoints are possible.
